hawkbit-http-security/src/main/java/org/eclipse/hawkbit/security/AbstractHttpControllerAuthenticationFilter.java

```diff
@@ -168,7 +168,7 @@ protected TenantSecurityToken createTenantSecruityTokenVariables(final HttpServl
 
     private TenantSecurityToken createTenantSecruityTokenVariables(final HttpServletRequest request,
             final String tenant, final String controllerId) {
-        final TenantSecurityToken secruityToken = new TenantSecurityToken(tenant, controllerId,
+        final TenantSecurityToken secruityToken = new TenantSecurityToken(tenant, null, controllerId, null,
```

Comment:
a second `TenantSecurityToken` constructor would probably make sense due the `tenant` and `tenantId` would probably never filled out together, right? Either one of these, so a constructor which offers `tenantId` and `controllerId` then we do not need to pass `null` values